ui/package.json

```diff
@@ -9,7 +9,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.1.14",
     "@kubernetes/client-node": "github:scality/kubernetes-client-javascript.git#browser-0.10.2-63-g579d066",
-    "@scality/core-ui": "github:scality/core-ui.git#17f9913ea6a068753b71d60711740c75c0232af7",
+    "@scality/core-ui": "github:scality/core-ui.git#v0.26.0",
```

Comment:
Let's just put the `package.json` and `package-lock.json` change in a separate comment.

The idea is to have a rather clean history.. thanks!